certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "type": "string"
+        },
+        "submissionReference": {
+          "description": "Sender's reference key for the document",
+          "type": "string"
+        },
+        "documentBase64": {
+          "description": "Gets or Sets DocumentBase64",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrder": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "version": {
+          "description": "Gets or Sets Version",
+          "type": "string"
+        },
+        "key": {
+          "format": "uuid",
+          "description": "Gets or Sets Key",
+          "type": "string"
+        },
+        "detail": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderDetail"
+        },
+        "products": {
+          "description": "Gets or Sets Products",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.ProductDetail"
+          }
+        },
+        "transactionParties": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TransactionParties"
+        },
+        "financial": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Financial"
+        },
+        "properties": {
+          "description": "Gets or Sets Properties",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Property"
+          }
+        },
+        "commitment": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderCommitment"
+        },
+        "policy": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderPolicy"
+        },
+        "parties": {
+          "description": "Gets or Sets Parties",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderParty"
+          }
+        },
+        "notes": {
+          "description": "Gets or Sets Notes",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderNote"
+          }
+        },
+        "documents": {
+          "description": "Gets or Sets Documents",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Document"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.PartiesDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "titleCompany": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "underwriter": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "escrowBank": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Loan": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.